UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.LoadTests/LiveLogs/ESSPublishedLogs.json

```diff
@@ -0,0 +1,5 @@
+[
+  {
+    "Comment": "Add your azure logs here"
+  }
+]
```

Comment:
Please add proper comment to clarify what content this file contains and for what purpose it get used.

Reply:
removed